frontend/service.go

```diff
@@ -19,6 +18,7 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/pkg/errors"
```

Comment:
Why is this change part of this PR? I went and looked at https://github.com/pkg/errors and noticed that `pkg/errors` was archived in 2021.